checkST2110.js

```diff
@@ -473,35 +605,43 @@ const extractMTParams = (sdp, params = {}) => {
   let isSkippedType = false;
   let streamCount = 0;
   let payloadType = -1;
-  for ( let x = 0 ; x < lines.length ; x++ ) {
+  let mediaType = -1;
+  for (let x = 0; x < lines.length; x++) {
     if (lines[x].startsWith('m=')) {
       let videoMatch = lines[x].match(videoPattern);
-      payloadType = videoMatch ? +videoMatch[4] : -1;
+      let audioMatch = lines[x].match(audioPattern);
+      if (videoMatch)
+        payloadType = +videoMatch[4];
+      if (audioMatch)
+        payloadType = +audioMatch[4];
       streamCount++;
       isSkippedType = false;
       continue;
     }
     if (lines[x].startsWith('a=rtpmap') && payloadType >= 0) {
       let rtpmapMatch = lines[x].match(rtpmapPattern);
+      if (rtpmapMatch)
+        mediaType = rtpmapMatch[2];
       if (rtpmapMatch && skipVideoTypes.includes(rtpmapMatch[2])) {
         isSkippedType = true;
         continue;
       }
     }
     if (lines[x].startsWith('a=fmtp') && payloadType >= 0 && !isSkippedType) {
       if (!fmtpPattern.test(lines[x])) {
+        errors.push(new Error(`Line ${x + 1}: fmpt ${lines[x]}has error`));
```

Comment:
Can we explain and format the error better here? We know the line starts with `a=fmtp` but that it doesn't meet the pattern. (At least, `fmtp` not `fmpt` and a space before `has`?)